### ISSUE TITLE
New Version 2.0: Downloaded Videos Audio-Only And Fast-Paced Playback

### ISSUE THREAD

**Issue by Airiset** [EngSci-Tools/Lecture-Downloader#1]
So far, the extension's repository has not been updated with the Version 2.0 changes. Maybe the GitHub repo is no longer active. Nevertheless, I decided to report this issue anyway.  

While using the new version (2.0) of the extension, I found some issues with downloaded videos. The two issues are as follows:
- The videos are audio-only. They have neither visuals nor thumbnails. 
- The audio is incredibly fast.

I am willing to send you the downloaded video using the old vs. the new version for diagnosis purposes. I do not post it directly here for both privacy and copyright purposes. Moreover, my current operating system is Windows 10.  I hope that I can be of help; This extension rocks!

P.S. I just noticed that this same issue was found by another user who posted it on Chrome Web Store reviews.

**Comment by Airiset** [comment]
Update:  The issues I previously described occurred when viewing the video in Windows Media Player. I tried slowing down the playback speed to see how much it had changed. The speed at which the audio sounded somewhat normal was x0.5 playback speed. However, the audio itself remained choppy. 

Moreover, I found something interesting. I tried playing the same video on VLC Media Player. The video (visuals) was working! The length of the video was the same as the original. However, the audio itself remained broken.
